generic-styles/index.less

```diff
@@ -139,6 +139,10 @@ body {
       display: inline;
     }
   }
+
+  .os-table td {
+    height: 3rem;
```

Comment:
Shouldn't we use `min-height` instead?

Reply:
nope, `height` for td is a `min-height` and `min-height` is not working :p
I was also confused :D